packages/app/src/cli/services/app/config/link.test.ts

```diff
@@ -1,13 +1,13 @@
-import link, {LinkOptions, emptyApp} from './link.js'
-import {saveCurrentConfig} from './use.js'
+import link, {LinkOptions} from './link.js'
```

Comment:
This is a big diff section -- its just asserting existing the result of `link` for lots of these tests, and moving the content assertion to the end (as its less actionable than the other assertions)